### ISSUE TITLE
Accessibility - Form control with "listbox" role has no associated label

### ISSUE THREAD

**Issue by kyjak** [carbon-design-system/carbon-for-ibm-dotcom#6115]
the label is missing on a listbox
reporting url https://ibmdocs-dev.mybluemix.net/docs/en/error
and the tool to check errors is "IBM Equal Access Accessibility Checker", plugin for chrome

`<div class="react-autosuggest__suggestions-container" role="listbox" id="react-autowhatever-1">`

**Comment by RobertaJHahn** [comment]
Kenny will review and document more details for the next review.

**Comment by kennylam** [comment]
Confirmed fix is needed for React version.

**Comment by kennylam** [comment]
@Praveen-sr This issue as well as #6116 and #6117 are ready for testing.
